content/en/docs/reference/pytorchjob/v1/pytorch.md

```diff
@@ -56,7 +56,7 @@ string
 <td>
 <code>metadata</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#objectmeta-v1-meta">
+<a href="https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta">
```

Comment:
Please can you also fix the link for the MPIJob: https://www.kubeflow.org/docs/reference/mpijob/v1alpha2/mpi/#github.com/kubeflow/mpi-operator/pkg/apis/kubeflow/v1alpha2.MPIJob

Reply:
@andreyvelich done